components/DynamicContent/src/index.scss

```diff
@@ -66,8 +96,18 @@
 }
 
 .denhaag-dynamic-content__card .denhaag-icon {
+  --denhaag-dynamic-content-card-icon-translate: translateX(-0.1875rem);
+
   align-self: end;
   color: var(--denhaag-dynamic-content-icon-color);
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```

Comment:
Target specific changes. Now you make the transition heavier for no reason.

Reply:
This is referencing more than one transitions:
![Screenshot 2022-11-29 at 17 14 45](https://user-images.githubusercontent.com/95216123/204583453-b4b59751-8f60-4d2b-a19b-208ca712d7a1.png)
![Screenshot 2022-11-29 at 17 15 04](https://user-images.githubusercontent.com/95216123/204583458-7ddf39eb-3320-4181-8820-6a9b8bacb5c6.png)

I cannot put these in the shorthand 'transition:' also its not possible to do twice this transition in the selector.

Reply:
done